src/2d/Camera2D.hpp

```diff
@@ -20,16 +20,22 @@ namespace ari::en
 		//! Destructor
 		virtual ~Camera2D() = default;
 
-
+		sx_vec2		Target,
+					Up,
+					Right;
 		float		AspectRatio;
```

Comment:
I think we also don't need this

Reply:
But how we should change the aspect of the camera? I mean when we want a 3:4 or 16:9 view...

Reply:
Aspect is width/height, in this case, xmag & ymag, Maybe you have to rename them to width & height